package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "inverted-index",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "gulp",
+    "test": "gulp test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/andela-ooyeniran/inverted-index.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/andela-ooyeniran/inverted-index/issues"
+  },
+  "homepage": "https://github.com/andela-ooyeniran/inverted-index#readme",
+  "dependencies": {
+    "browser-sync": "^2.18.6",
```

Comment:
You have dependencies duplicate, you may want to look at them and determine the right place where each should be